modelplugin/plproxy-1.0.0/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/onosproject/config-models/modelplugin/plproxy-1.0.0
+
+go 1.15
```

Comment:
should be 1.16

Reply:
thanks for catching it i don't how did that happened